10.py

```diff
@@ -0,0 +1,21 @@
+import pandas as pd
+import random
+
+lst = ['robot'] * 10
+lst += ['human'] * 10
+random.shuffle(lst)
+
+data = pd.DataFrame({'whoAmI': lst})
+print('Исходный DataFrame:\n', data.head())
+
+# создаем список уникальных значений в столбце "whoAmI"
+unique_vals = data['whoAmI'].unique()
+
+# создаем пустой DataFrame с столбцами для каждого уникального значения
+one_hot_data = pd.DataFrame(columns=unique_vals)
```

Comment:
Можно просто добавить новые столбцы к существующей таблице
```suggestion
data[unique_vals] = 0
```